cloud

```diff
@@ -13,6 +13,7 @@
 set -eEuCo pipefail
 shopt -s extglob
 
+declare -r TMPDIR_ORIG=${TMPDIR:-}
```

Comment:
Please add a comment above this line, like I had in my example for this variable.

Reply:
done